card-service/Dockerfile

```diff
@@ -14,6 +14,7 @@ COPY requirements.txt .
 RUN python -m pip install -r requirements.txt
 
 WORKDIR /app
+COPY . /app
```

Comment:
remove this because omitting it is necessary for hot reloading (i think) while using docker compose

Reply:
done. clarify